README.md

```diff
@@ -20,10 +20,12 @@ setup:
 - Install Docker, and make sure that the current user is in the `docker` group.
 - Install jq
 - (optionally) Install tmux -- this only needed if you want to use it.
+- Install Docker Buildx (required for building Docker images)
```

Comment:
I suggest moving L23 below L20 to cluster Docker-related information more tightly.